specs/phase0/p2p-interface.md

```diff
@@ -644,39 +650,27 @@ ENRs MUST carry a generic `eth2` key with an 16-byte value of the node's current
 |:-------------|:--------------------|
 | `eth2`       | SSZ `ENRForkID`        |
 
-First we define `current_fork_data` as the following SSZ encoded object
-
-```
-(
-    current_fork_version: Version
-    genesis_validators_root: Root
-)
-```
-
-where
-
-* `current_fork_version` is the fork version at the node's current epoch defined by the wall-clock time (not necessarily the epoch to which the node is sync)
-* `genesis_validators_root` is the static `Root` found in `state.genesis_validators_root`
-
 Specifically, the value of the `eth2` key MUST be the following SSZ encoded object (`ENRForkID`)
 
 ```
 (
-    current_fork_digest: ForkDigest
+    fork_digest: ForkDigest
     next_fork_version: Version
     next_fork_epoch: Epoch
 )
 ```
 
 where the fields of `ENRForkID` are defined as
 
-* `current_fork_digest` is `ForkDigest(hash_tree_root(current_fork_data)[:4])`
+* `fork_digest` is `ForkDigest(compute_fork_data_root(current_fork_version, genesis_validators_root)[:4])` where
```

Comment:
`Status`  message needs updating as well, to use `fork_digest`

Reply:
Does it?

I would tend to agree, but it does change fundamental functionality here. `fork_digest` is in relation to wall-clock time whereas `Status.head_fork_version` was in relation to the head of your current chain (so might be behind while you're syncing).

The granularity of having Status version wrt your current chain might allow for marginally more optionality in syncing sections of a chain with a peer with mismatched forks in the future. Whereas straight `fork_digest` wrt wall-clock time will be much cleaner to handle.

I lean toward the clean `fork_digest` for cleanliness and not trying to optimize for a fringe case

Reply:
added in fork_digest